### DISCUSSION TITLE
Performance comparison of similar libs of php

### DISCUSSION THREAD

**Discussion by guanguoliang** [qax-os/excelize#1167] [Q&A]
I see the doc of https://xuri.me/excelize/en/performance.html. The performance of php only use 33s to generate 102400 * 50 cells? I use the latest version of PhpSpreadsheet of 1.22 to generate the same excel  with my personal computer which has the same Mackbook, and the cost time was 218.02s, much more than 33s. So I doubt the data of the comparison

**Comment by xuri** [comment]
I have updated the benchmark report for the latest version of the Excelize library, and you can find out the benchmark script in [this repo](https://github.com/xuri/excelize-benchmark/tree/2f1b4a646dd36d3cc03febe275d0d7526cdfea5f/etc).
